cluster/examples/kubernetes/ceph/admission-controller/hack/webhook-create-signed-cert.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+# Original script found at: https://github.com/morvencao/kube-mutating-webhook-tutorial/blob/master/deployment/webhook-create-signed-cert.sh
+
+set -e
```

Comment:
This is a big enough feature that I'd ask that we have a [design doc](https://github.com/rook/rook/tree/master/design/ceph) to describe the overall approach. For example, here are some questions that need to be answered:
- How will the admission controller be deployed? An operator side car?
- Certificate management
   - Deploying in a test env can work with a self signed cert?
   - A cert must be deployed for this to work in production? How is it managed?
- Can we always deploy the admission controller? Or should it be optional?

Reply:
https://github.com/rook/rook/pull/5151